src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -0,0 +1,160 @@
+import { BasicField } from '../../../../types'
+import {
+  ProcessedCheckboxResponse,
+  ProcessedTableResponse,
+} from '../submission.types'
+
+import {
+  EmailAutoReplyField,
+  EmailDataForOneField,
+  EmailJsonField,
+  ResponseFormattedForEmail,
+} from './email-submission.types'
+
+/**
+ * Determines the prefix for a question based on whether it is verified
+ * by MyInfo.
+ * @param {Object} response
+ * @param {Set} hashedFields Hash for verifying MyInfo fields
+ * @returns {String}
+ */
+const getMyInfoPrefix = (
+  response: ResponseFormattedForEmail,
+  hashedFields: Set<string>,
+): string => {
+  return !!response.myInfo?.attr && hashedFields.has(response._id)
+    ? '[MyInfo] '
```

Comment:
nit: should all the prefixes be a constant declared somewhere instead of an inlined string? so other devs can see at a glance all the prefixes/how to change

```ts
const MYINFO_PREFIX = '[MyInfo] '
```
... etc